File: Completed/js/addTwoDigits/addTwoDigits.js
//STRING WAY
let addTwoDigitsStr = (n) => n.toString().split("").reduce((i, j) => parseInt(i) + parseInt(j));

//MATH WAY
function addTwoDigitsMath(n) {
    let sum = 0;
    while (n % 10 > 0) {
        sum += n % 10;
        n = Math.floor(n / 10)
    }
    return sum;
}

console.time("String Function: ");
console.log(addTwoDigitsStr(29));
console.timeEnd("String Function: ");
console.time("Math Function: ");
console.log(addTwoDigitsMath(29));
console.timeEnd("Math Function: ");

File: Completed/js/addTwoDigits/addTwoDigits.py
import math


def addTwoDigitsStr(n):
    return sum([int(i) for i in list(str(n))])


def addTwoDigitsMath(n):
    num = 0
    while(n % 10 > 0):
        num += n % 10
        n = math.floor(n/10)
    return num


print(addTwoDigitsStr(29))
print(addTwoDigitsMath(29))
